src/components/ChampionItem/ChampionItem.js

```diff
@@ -33,49 +32,43 @@ const useStyles = makeStyles((theme) => ({
 
 const ChampionItem = (props) => {
     const classes = useStyles();
-    const { champions } = props;
-    if (!champions || champions.length === 0) return <p>No champions, sorry</p>;
 
     return (
         <div className={classes.root}>
-            {champions.map((champion, key) => {
-                return (
-                    <Paper className={classes.paper}>
-                        <Grid container spacing={2}>
-                            <Grid item>
-                                <ButtonBase className={classes.image}>
-                                    <img className={classes.img} alt="complex" src={champion.image_url} />
-                                </ButtonBase>
-                            </Grid>
-                            <Grid item xs={12} sm container>
-                                <Grid item xs container direction="column" spacing={2}>
-                                    <Grid item xs>
-                                        <Typography gutterBottom variant="subtitle1">
-                                            {champion.name}
-                                        </Typography>
-                                        <Typography variant="body2" gutterBottom>
-                                            ARMOR: {champion.armor}
-                                        </Typography>
-                                        <Typography variant="body2" color="textSecondary">
-                                            ATTACK DAMAGE: {champion.attackdamage}
-                                        </Typography>
-
-                                    </Grid>
+            <Paper className={classes.paper}>
```

Comment:
add key={key} here. Each list item should have an unique identifier.

Reply:
<img width="653" alt="Screenshot 2021-05-06 at 15 22 37" src="https://user-images.githubusercontent.com/29281918/117297464-ee2d7180-ae7e-11eb-8125-323a3a23de6e.png">

Added key={key} in ChampionList.js because in ChampionItem.js on Paper it doesn't work.
Thanks.